content/_about.es.md

```diff
@@ -0,0 +1,19 @@
++++
+date = "2017-03-07T18:15:51-03:00"
+title = "Sobre nosotros"
+url = "sobre-nosotros"
++++
+
+## Sobre nosotros
+
+En Uruguay existen muchas personas trabajando con JS al primer nivel pero a veces fallamos en comunicarle al resto del mundo lo que estamos haciendo. La mayoría de los desarrolladores fallamos en contar nuestros logros mientras algunos otros ni siquiera sabemos que hacerlo es beneficioso para el desarrollo de su carrera.
+
+Creamos `☀️.js.uy` para promover las actividades y potenciar a los integrantes de la comunidad JS en Uruguay en pos de hacer crecer la misma.
+
+Creemos que ayudando a la difusión a nuestras actividades vamos a tener un impacto positivo en las carreras de las personas. Ayudándolos a tener más notoriedad en la comunidad internacional y que esto les permita contactar con mejores clientes, mejorar sus sueldos y elegir los proyectos que más les interesen.
```

Comment:
"personas, ayudándolas..." ya que se continúa la misma idea, siendo "personas" son "las", se podría cambiar a "desarrolladores" de lo contrario.
"mayor visibilidad" en lugar de "más"?
"visibilidad en la comunidad, favoreciendo su acceso a proyectos más interesantes."?
Son sugerencias, feel free to dismiss...